src/style.css

```diff
@@ -0,0 +1,134 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+.html {
+  font-family: Avenir, sans-serif;
+}
+
+.label.searchbar {
+  width: 100%;
+  height: 70px;
+}
+
+.searchbar--big {
+  position: relative;
+  width: 100%;
+  height: 70px;

```

Comment:
Remove `height` property. Your form height will be calculated automatically based on the `input` height.